Sources/Target/Linux/Process.cpp

```diff
@@ -215,19 +215,54 @@ ErrorCode Process::wait() {
 
     switch (_currentThread->_stopInfo.event) {
     case StopInfo::kEventNone:
-      // If the thread is stopped for no reason, it means the debugger (ds2)
-      // sent a SIGSTOP to it while it was already stopped for another reason,
-      // the SIGSTOP was queued, and now we're getting this notification. We
-      // have to ignore this and just let the thread continue.
-      // If `stepping` is true, it means the thread was actually being
-      // single-stepped before stopping, so instead of doing a `resume()`, we
-      // have to do a new `step()`.
-      if (stepping) {
+      // If the _stopInfo.event is kEventNone it means the thread is either
+      // 1. stopped because a sibling thread hit a breakpoint and ds2
+      //    sent a SIGSTOP to pause this thread while the breakpoint is
+      //    evaluated. In this case we just call continue.
```

Comment:
`s/call continue/resume the thread/g`

There are other instances of this below.